libs/component-sample/src/lib/feature/side-navigation-sample/side-navigation-sample.component.ts

```diff
@@ -83,6 +99,19 @@ export class SideNavigationSampleComponent implements AfterViewInit, OnInit {
     }
     this.sideNav.select(this.selectedId);
     this.change.detectChanges();
+
+    this.activeRoute.queryParams.subscribe(queryParams => {
+      if (queryParams.item) {
+        this.pageHeader = queryParams.item;
+        if (this.model.navigationLinks) {
+          this.findItemByQueryString(this.model.navigationLinks);
+          this.sideNav.select(this.selectedId);
+        }
+      } else {
+        this.pageHeader = 'Unknown';
```

Comment:
The default of Unknown could be a bad experience for users.

Options I can think of:
1. Make item required
2. Allow the H1 to be optionally displayed when item is set
3. Set a more friendly default value

Reply:
This is just the sample page

Reply:
That is me just taking the query string value and displaying it